python/ray/train/tensorflow/tensorflow_predictor.py

```diff
@@ -65,27 +61,28 @@ def __init__(
                     "`batch_predictor.predict(ds, num_gpus_per_worker=1)` to "
                     "enable GPU prediction."
                 )
-
-        if model_weights is not None:
-            self._model.set_weights(model_weights)
         super().__init__(preprocessor)
 
     def __repr__(self):
-        fn_name = getattr(self.model_definition, "__name__", self.model_definition)
+        fn_name = getattr(self._model, "__name__", self._model)
+        fn_name_str = ""
```

Comment:
@amogkam @bveeramani let me know if this is OK. This is mostly to satisfy the length requirement. 
It would be something like `tf.keras.Sequential object` with some cut-off.